File: programmers/lv0/코딩기초트레이닝/출력/대소문자바꿔서출력하기/Solution.go
package main

import (
	"fmt"
	"unicode"
)

func main() {
	var s1 string
	fmt.Scan(&s1)
	for _, c := range s1 {
		switch {
		case unicode.IsUpper(c):
			fmt.Printf("%c", unicode.ToLower(c))
		case unicode.IsLower(c):
			fmt.Printf("%c", unicode.ToUpper(c))
		default:
			fmt.Printf("%c", c)
		}
	}
	fmt.Println()
}

File: programmers/lv0/코딩기초트레이닝/출력/대소문자바꿔서출력하기/Solution.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    string str;
    cin >> str;
    for (int i = 0; i < str.length(); i++) {
        char c = str[i];
        if (isupper(c)) {
            cout << (char)tolower(c);
        } else {
            cout << (char)toupper(c);
        }
    }
    return 0;
}
